frontend/main/src/components/PostsCards.tsx

```diff
@@ -7,102 +7,111 @@ export default function PostsCards({ posts }: { posts: Post[] }): JSX.Element {
+                            height="270"
+                            layout="responsive"
+                            className="rounded-md rounded-b-none cursor-pointer"
+                          />
+                        </>
+                      ) : (
+                        <div
+                          className="relative"
+                          style={{ paddingBottom: '56.25%' }}
+                        >
+                          <div className="absolute flex items-center flex-auto w-full h-full rounded-t-md bg-primary-900 dark:bg-primary-900">
+                            <AJPrimary className="w-full h-full p-4" />
+                          </div>
+                          {post.type === 'course' ? (
+                            <p className="absolute px-4 py-2 rounded-full bg-secondary-600 dark:bg-secondary-600 -bottom-12 -right-8 text-basics-50 dark:text-basics-50">
+                              Start now
+                            </p>
+                          ) : (
+                            <></>
+                          )}
                         </div>
-                        {post.type === 'course' ? (
-                          <p className="absolute px-4 py-2 rounded-full bg-secondary-600 dark:bg-secondary-600 -bottom-12 -right-8 text-basics-50 dark:text-basics-50">
-                            Start now
-                          </p>
-                        ) : (
-                          <></>
-                        )}
-                      </div>
-                    )}
-                  </a>
-                </Link>
+                      )}
+                    </a>
+                  </Link>
 
-                <section className="grid h-full gap-2 p-4">
-                  <h3 className="font-sans text-lg tracking-wide text-basics-900 text-bold">
-                    <Link href={`/${post.type}/${post.slug}`}>
-                      <a>{post.title}</a>
-                    </Link>
-                  </h3>
-                  <p className="text-sm font-hairline text-basics-900">
-                    {post.excerpt}
-                  </p>
-                  <div>
-                    {post.authors?.map((author, i) => (
-                      <h4 key={i} className="font-sans text-lg">
-                        {author.displayName}
-                      </h4>
-                    ))}
-                  </div>
-                  <div className="grid self-end justify-start grid-cols-2 gap-2 font-bold tracking-wider">
-                    {post.type === 'course' && (
-                      <>
-                        {post.accessSettings?.accessMode === 'closed' ? (
-                          <>
-                            <p className="px-4 py-2 text-sm rounded-full justify-self-start bg-primary-900 dark:bg-primary-900 text-basics-50 dark:text-basics-50">
-                              Paid
+                  <section className="grid h-full grid-cols-1 gap-2 p-4">
```

Comment:
I believe in light mode the text is not showing correctly.
![image](https://user-images.githubusercontent.com/3102249/112146747-53b8ed80-8bb2-11eb-9bb0-f3b7ab554ec5.png)